packages/extension-chakra-storefront/src/components/footer/index.jsx

```diff
@@ -192,37 +174,26 @@ const Footer = ({...otherProps}) => {
 export default Footer
 
 const Subscribe = ({...otherProps}) => {
-    const styles = useStyles()
+    const recipe = useSlotRecipe({key: 'footer'})
+    const styles = recipe()
     const intl = useIntl()
     return (
-        <Box {...styles.subscribe} {...otherProps}>
+        <Box css={styles.subscribe} {...otherProps}>
             <Heading as="h1" {...styles.subscribeHeading}>
```

Comment:
Are we missing using the css prop?
```suggestion
            <Heading as="h1" css={styles.subscribeHeading}>
```

Reply:
Good catch, fixed